contracts/BinaryOptionMarketData.sol

```diff
@@ -108,6 +108,11 @@ interface IBinaryOptionMarket {
         uint finalPrice;
     }
 
+    struct OraclePriceAndTimestamp {
```

Comment:
This struct doesn't actually exist in the `BinaryOptionMarket`, so it should actually be declared in the `BinaryOptionMarketData` contract.